docs/sop/PRODUCTION_DEPLOY.md

```diff
@@ -0,0 +1,344 @@
+for checking in the latter live deployment steps.
+
+Change directory to the `envs` folder:
+```
+$ cd ops/infrastructure/envs
+```
+
+Change directory to the `live` directory:
+```
+$ cd live
+```
+
+Copy terraform files to `live` environment:
+```
+$ ../../../scripts/tf_init.sh --project gigascience/upstream/gigadb-website --env live
+
+You need to specify an AWS region:
+ap-east-1
+
+You need to specify the path to the ssh private key to use to connect to the EC2 instance: 
+~/.ssh/id-rsa-aws-hk-gigadb.pem
+
+You need to specify your GitLab username:
+pli888 | rijam | kencho51
+
+You need to specify a backup file created by the files-url-updater tool:
+../../../../gigadb/app/tools/files-url-updater/sql/gigadbv3_20210929_v9.3.25.backup
+```
+
+Use Gigadb AWS IAM user account to provision production staging server:
+```
+$ terraform show # will show all the existing resources, which should be the same as the terraform state file `live_infra` from gitlab.
+$ terraform plan # terraform is idempotent and should not try to create new instances for already existing upstream staging, unless the new instance is expected to create.
+$ terraform apply # will make changes to the existing infrastructure and update the terraform state file, input `yest` if the changes are expected to make.
+$ terraform refresh
+```
+
+Copy ansible files into `live` environment:
+```
+$ ../../../scripts/ansible_init.sh --env live
+```
+
+Install third party Ansible roles:
+```
+$ ansible-galaxy install -r ../../../infrastructure/requirements.yml
+```
+
+Provision RDS via bastion server:
+```
+$ env OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES ansible-playbook -i ../../inventories bastion_playbook.yml --extra-vars="gigadb_env=staging"
+```
+
+Enable cronjob for periodically resetting the database:
+```
+$ env OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES ansible-playbook -i ../../inventories bastion_playbook.yml -e "reset_database_cronjob_state=present" --extra-vars="gigadb_env=staging"
+```
+
+Enable cronjob for periodically creating a database dump and storing it in S3:
+```
+$ env OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES ansible-playbook -i ../../inventories bastion_playbook.yml -e "upload_database_backup_to_S3_cronjob_state=present" --extra-vars="gigadb_env=staging"
+```
+
+To confirm the cron jobs have been created, log in with ssh on bastion, and run 
+`crontab -l` - there should be a cronjob to reset the database daily at 10:05am 
+UTC (after the CNGB backup of day before is made available).
+```
+# Get public ip of bastion server
+$ terraform output
+
+# ssh into it
+$ ssh -i ~/.ssh/id-rsa-aws-hk-gigadb.pem centos@<bastion public ip>
+```
+
+Provision web application server:
+```
+$ TF_KEY_NAME=private_ip OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES ansible-playbook -i ../../inventories webapp_playbook.yml --extra-vars="gigadb_env=staging"
```

Comment:
**typo**: we are in the section for provisioning live, but  I see `--extra-vars="gigadb_env=staging"` here
**suggestion**: replace with `--extra-vars="gigadb_env=live"`